code/_helpers/mobs.dm

```diff
@@ -91,7 +91,7 @@
 	var/starttime = world.time
 	. = 1
 	while (world.time < endtime)
-		sleep(1)
+		stoplag(1)
```

Comment:
These changes are unrelated to the PR.

Reply:
They're documented in PR description.

Reply:
I don't really get it either; it's one thing to stoplag after tick checks, but just a straight stoplag like this feels expensive and pointless. I don't know enough about the internal scheduling to make fully informed comments, though.